src/MergeContext.js

```diff
@@ -527,7 +527,7 @@ class MergeContext {
 
     _prOpen() { return this._pr.state === 'open'; }
 
-    _prBody() { return this._pr.body.replace(/\r+\n/g, '\n'); }
+    _prBody() { return this._pr.body ? this._pr.body.replace(/\r+\n/g, '\n') : ""; }
```

Comment:
I would worry that some non-null pr.body values/text might be [falthy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy). Please use an explicit comparison with null (and, if needed, undefined).

Reply:
Done.